types/node-statsd/index.d.ts

```diff
@@ -0,0 +1,171 @@
+// Type definitions for node-statsd 0.1
+// Project: https://github.com/sivy/node-statsd
+// Definitions by: Alex Turek <https://github.com/alexturek>
+//                 Convoy <https://github.com/convoyinc>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// Comments copied with modifications from
+// https://github.com/sivy/node-statsd/blob/master/lib/statsd.js
+/// <reference types="node" />
+
+import * as dgram from 'dgram';
+
+declare module "node-statsd" {
```

Comment:
do not create a module declaration, just export the contents directly from this file. i.e.:

```ts
export interface StatsDConfig {
...
}
```

instead of:
```ts
declare module "node-statsd" {
    interface StatsDConfig {
      ....
   }
}
```